.github/workflows/ci.yml

```diff
@@ -55,7 +55,7 @@ env:
   RUSTFLAGS: -D warnings
   RUSTDOCFLAGS: -D warnings
   WASM_BINDGEN_TEST_TIMEOUT: 300 # 5 minutes
-  CACHE_SUFFIX: c # cache busting
+  CACHE_SUFFIX: d # cache busting
```

Comment:
Not strictly necessary, but I thought I had a cache bug, and it doesn't hurt to make sure the caches are clear every once and again